requirements-dev.txt

```
@@ -9,7 +9,7 @@ bandit[toml] >=1.0, <2.0
 black >=23.0, <24.0
 flake8 >=7.0, <8.0
 flake8-bandit >=4.0, <5.0
-flake8-bugbear >=23.0, <24.0
+flake8-bugbear >=23.0, <25.0
```

```suggestion
flake8-bugbear >=24.0, <25.0
```

Also change `.pre-commit-config.yaml` accordingly.